src/fight/mod.rs

```diff
@@ -0,0 +1,1058 @@
+//! # Fight Module
+//!
+//! This module contain all the functions to emulate a fight between 2 Regiment
+//! This may evolve to multiple Regiment against multiple Regiment
+
+mod global_values;
+use crate::{model, regiment, roll};
+
+fn get_nb_attacks(regiment_attacking: &regiment::Regiment) -> usize {
+    regiment_attacking.get_model().get_stats().get_attack() * regiment_attacking.get_cols()
+        + ((regiment_attacking.get_model().get_stats().get_attack() * regiment_attacking.get_cols()
+            / 2) as usize)
+}
+
+fn get_nb_wounds(nb_attacks: usize, to_hit: usize, to_wound: usize) -> usize {
+    let nb_hit: usize = (((nb_attacks * (global_values::DEFAULT_DICE - to_hit + 1))
+        / global_values::DEFAULT_DICE) as f64)
+        .ceil() as usize;
```

Comment:
We can it doesn't really matter to me.

Reply:
If it doesn't provide way more accurate result then maybe it's not worth the trouble, as long as we are not wrong mathematically in the various unit tests

Reply:
in fact round is more accurate because "ceil" round to the sup number while "round" round to the nearest number
technically round > ceil @H4ss

Reply:
Done

Reply:
Yes. But I was saying if it was necessary to change every ceil to round in this PR because we lack accuracy or if we should start using it from now on as a conventional way to deal with the decimal point rounding ? I'll understand in either way, i'm asking for you opinion ;)